### PR TITLE
Ft2023-342: Creating and customising Route system for Custom module routing_system. 

### PR DESCRIPTION
**Pull request by Souvik-Inno**
Create a route to define a controller and make the route accessible only by admin and editor users
  - Update the same route and make it accessible to those who have “access the custom page” permission.
  - Alter the existing route and remove the role access of the editor. [Not from routing.yml file]
  - Add a custom access check if the user can view the page or not.

Create a custom route /campaign/value/2. Here, the third parameter is dynamic. Now fetch the third value from the route,i.e., 2.  in the Controller

### CODE REVIEW

File: web/modules/custom/routing_system/routing_system.info.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+name: Routing System
+description: Routing system which uses dynamic routing
+package: Custom module
+type: module
+core_version_requirement: ^10.0
```
Comment (vivekpan27) [inline web/modules/custom/routing_system/routing_system.info.yml:5]: 10.0?

File: web/modules/custom/routing_system/src/Controller/RouteController.php [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   *   Array to render page.
+   */
+  public function content() {
+    if ($this->currentUser->hasPermission('Routing Permission')) {
+      return [
+        '#markup' => $this->t('You have a granted access to the page.'),
+      ];
+    }
+    return [
+      '#markup' => $this->t('You dont have access to the page.'),
```
Comment (vivekpan27) [inline web/modules/custom/routing_system/src/Controller/RouteController.php:53]: Better to return 403 response.

Comment (Souvik-Inno) [reply web/modules/custom/routing_system/src/Controller/RouteController.php]: Okay, changed.

File: web/modules/custom/routing_system/routing_system.info.yml (hunk 1 above)
Comment (Souvik-Inno) [reply web/modules/custom/routing_system/routing_system.info.yml]: Okay changed to 10

**Review by vivekpan27 — APPROVED** [review]

**Pull request merged**